larq_compute_engine/tflite/tests/BUILD

```diff
@@ -5,7 +5,7 @@ package(
 
 cc_test(
     name = "bconv2d_test",
-    size = "small",
+    size = "medium",
```

Comment:
These tests now take quite a while to run (~150 seconds on my 8 core machine) so I had to change this to avoid them timing out.